pkg/ts/render/Composer.ts

```diff
@@ -90,6 +91,8 @@ export class Composer extends EventEmitter {
   generateSlot(id, template, parent): Slot {
     return null;
   }
+  maybeReattachSlot(slot, container) {
```

Comment:
implementation in xen-composer:
```
  maybeReattachSlot(slot, container) {
    if (!slot.root.isConnected) {
      this.findContainer(container).appendChild(slot.root.host);
    }
  }
```

Reply:
Ahhhh ... nice!

Reply:
Probably we should be prepared for 'findContainer' to fail?

Reply:
good point - made the change and created the corresponding PR:
https://team-review.git.corp.google.com/c/arcs-team/arcsjs-apps/+/1512785